src/test/java/guitests/BoardTests.java

```diff
@@ -0,0 +1,70 @@
+package guitests;
+
+
+import javafx.scene.input.KeyCode;
+import javafx.scene.input.KeyCodeCombination;
+import javafx.scene.input.KeyCombination;
+import javafx.stage.Stage;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.Before;
+import org.junit.Test;
+import prefs.Preferences;
+import ui.TestController;
+import ui.UI;
+import ui.issuepanel.PanelControl;
+import util.PlatformEx;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.loadui.testfx.Assertions.assertNodeExists;
+import static org.loadui.testfx.controls.Commons.hasText;
+
+public class BoardTests extends UITest {
+    PanelControl panelControl;
+    @Before
+    public void setup() {
+        UI ui = TestController.getUI();
+        panelControl = ui.getPanelControl();
+
+        Preferences testPref = UI.prefs;
+
+        List<String> boardNames = testPref.getAllBoardNames();
+        boardNames.stream().forEach(testPref::removeBoard);
+    }
+
+    @Test
+    public void duplicateNameTest() {
+        // Save the current board
+        clickMenu("Boards", "Save as");
+        click("OK");
```

Comment:
Can add a comment above this line that `board name will default to "New Board"` so that there is no confusion on what name the board is saved as?